src/libs/actions/Policy/Policy.ts

```diff
@@ -4421,6 +4435,23 @@ function enablePolicyAutoReimbursementLimit(policyID: string, enabled: boolean)
     });
 }
 
+function addNewCompanyCardsFeed(policyID: string, feedType: string, feedDetails: string) {
```

Comment:
Why are these in Policy.ts? I think we could have created something like CompanyCards.ts actions file for these actions

Reply:
I can prepare a follow-up and move all of the company cards related methods there!

Reply:
Follow uo PR is ready: https://github.com/Expensify/App/pull/51080
cc @allgandalf